code/modules/events/ghost_role/sentient_disease.dm

```diff
@@ -9,6 +9,26 @@
 	min_wizard_trigger_potency = 4
 	max_wizard_trigger_potency = 7
 
+/datum/round_event_control/sentient_disease/can_spawn_event(players_amt, allow_magic)
+	. = ..()
+	if(!.)
+		return .
+
+	var/host_counter = 0 //Counts how many potential hosts we have
+
+	//If 75% of the crew are potential hosts, we send in a sentient disease.
+	for(var/mob/living/carbon/human/candidate in shuffle(GLOB.player_list))
+		if(candidate.stat == DEAD)
+			continue
+		if(!(candidate.mind.assigned_role.job_flags & JOB_CREW_MEMBER)) //We're only checking crew here.
+			continue
+		if(length(candidate.diseases))
+			continue
+		host_counter++
+
+	if(((host_counter / length(GLOB.player_list)) * 100) < 75) //If under 75% of the crew is infected with something or ineligible, maybe it's not the best time to introduce a sentient disease.
```

Comment:
This is also a duplicate of the last comment, would rather you remove the previous one